package.json

```diff
@@ -47,6 +47,7 @@
     "@walletconnect/types": "^2.17.2",
     "bech32": "^2.0.0",
     "bignumber.js": "^9.1.2",
+    "buffer": "^6.0.3",
```

Comment:
Can we do without this dependency?

```ts
const HEX_STRINGS = '0123456789abcdef';

export function toHex(bytes: Uint8Array): string {
    return Array.from(bytes)
        .map((b) => HEX_STRINGS[b >> 4] + HEX_STRINGS[b & 15])
        .join('');
}
```

Reply:
yes we can. missed the presence of that.